Trees/deserialize_serialize_binary_tree.py

```diff
@@ -0,0 +1,37 @@
+class Solution():
+  '''
+  this problem shows how to convert a binary tree to serialized form which can make it easier to transfer/write to other formats.
+  The method to deserialize is given as well
+  '''
+  def serialize(self, root) :
+    nodes = [] # the 
+
+    def dfs(node):
+      if not node:
+        return "N"
+      #preorder addition of nodes to our list
+      nodes.append(node.value)
```

Comment:
this should be appending the string value of the node

Reply:
true